spec/support/features/in_person_helper.rb

```diff
@@ -205,32 +186,36 @@ def complete_entire_ipp_flow(user = user_with_2fa, tmx_status = nil, same_addres
   end
 
   def expect_in_person_step_indicator_current_step(text)
-    expect_in_person_step_indicator
-    expect_step_indicator_current_step(text)
-  end
-
-  def expect_in_person_step_indicator
     # Normally we're only concerned with the "current" step, but since some steps are shared between
     # flows, we also want to make sure that at least one of the in-person-specific steps exists in
     # the step indicator.
     expect(page).to have_css(
       '.step-indicator__step',
       text: t('step_indicator.flows.idv.find_a_post_office'),
     )
+    expect_step_indicator_current_step(text)
   end
 
-  def make_pii(same_address_as_id: 'true')
+  def build_pii_before_state_id_update(same_address_as_id: 'true')
     pii_from_user[:same_address_as_id] = same_address_as_id
```

Comment:
What about `build_pii_before_state_id_submission` or `build_pii_with_state_id`? You don't need to change this if you prefer your method name.

Reply:
I prefer `build_pii_before_state_id_update`.  I think it makes it clearer that this helper method is used after the user, while on the verify info page, chooses to update what information they entered on the state id page.